Cluster.WebCrawler/src/WebCrawler.CrawlService/CrawlService.cs

```diff
@@ -15,7 +15,7 @@ public bool Start(HostControl hostControl)
 
         public bool Stop(HostControl hostControl)
         {
-            ClusterSystem.Terminate();
+            CoordinatedShutdown.Get(ClusterSystem).Run();
```

Comment:
Should add a `.Wait(TimeSpan.FromSeconds(5))` to each of these, so we can force the process wait while the shutdown procedure runs